test/playwright/utils/page.ts

```diff
@@ -0,0 +1,13 @@
+import type { Page } from '@playwright/test'
+
+export const removeHiddenOverflow = async (page: Page) => {
```

Comment:
What is the reason for this?

Reply:
If you don't remove this you can't take full-page screenshots :disappointed: There's something about the overflow-hidden stuff that causes this. I don't understand why we need it, it seems related to the sticky header, but I'm not sure. @zackkrida said he might have a solution for removing it soon?

You can see the problem here: https://try.playwright.tech/?l=javascript&s=zbv3d2d

Note that even though the screenshot is for a "full page" it doesn't actually take the full page (because the overflow is hidden :upside_down_face:)

Reply:
Yes, I'll make an issue for the necessary layout refactoring today!